lib/elixir/lib/calendar/time.ex

```diff
@@ -402,16 +402,16 @@ defmodule Time do
   ## Examples
 
       iex> Time.to_seconds_after_midnight(~T[23:30:15])
-      84615
+      {84615, 0}
       iex> Time.to_seconds_after_midnight(~N[2010-04-17 23:30:15.999])
-      84615
+      {84615, 999000}
 
   """
   @doc since: "1.11.0"
   @spec to_seconds_after_midnight(Calendar.time()) :: integer()
```

Comment:
```suggestion
  @spec to_seconds_after_midnight(Calendar.time()) :: {integer(), integer()}
```

Reply:
yep, remembered that when doing the other PR, will update